### ISSUE TITLE
No name and role are defined for "Icons" present in Project screen: A11y_.NET 9 template_Projects_Name Role Value

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27432]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG4.1.2; #A11ySev1; #iOS; #SH_.NET9template_iOS_Jan2025; #Voiceover; #.NET Core; #Device; #screen reader; #NameRoleValue; #AINotInscope; #iphone;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2
Screen reader: Voiceover

### Repro Steps:

1. Turn on Voiceover and Install and open ".NET 9 template" app.
2. "Home Screen" will open. Swipe till Hamburger menu and double tap on it, Projects options will screen will open.
3. Swipe to any project and double tap to activate it.
4. Project screen will open. Swipe till Icon section.
5. Observe that No name and role is defined for Icons present in Project screen

### Actual Result:
No name and role are not defined for "Icons" present in Project screen.

### Expected Result:
Appropriate name and role should be defined for "Icons" present in Project screen

### User Impact:
Users who rely on screen readers will not receive information about the icons.

### Attachment:

https://github.com/user-attachments/assets/70504759-3181-485c-ab81-945ea9f61405

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:03-05-25; #A11yFixed;

Verified the bug in environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment

![Image](https://github.com/user-attachments/assets/cf91fee7-d829-4cbf-abab-b8c81cea2d83)
